### DISCUSSION TITLE
make rel attibute for links optional

### DISCUSSION THREAD

**Discussion by herzkerl** [outline/outline#3237] [Ideas]
When copying from Outline, every link tag has the `rel` attribute: `rel="noopener noreferrer nofollow“`. While this might be preferable for most situations, we currently use Outline for our Podcasts' show notes and therefore would like not to use `nofollow`.

I’m not sure about potential shortcomings, but I hope there could be a way to make that attribute optional when copying.

Let me know what you think. 🙏🏻

**Comment by tommoor** [comment]
What are you pasting into? I'm not sure this attribute makes sense to be optional but it could potentially be removed from the clipboard, I don't quite know the complexity of that yet.
